src/Core/Components/NavMenu/FluentNavMenu.razor.cs

```diff
@@ -46,6 +46,12 @@ public partial class FluentNavMenu : FluentComponentBase
     [Parameter]
     public bool Collapsible { get; set; }
 
+    /// <summary>
+    /// Gets or sets whether a menu with all child links is shown for <see cref="FluentNavGroup"/>s when the navigation menu is collapsed.
+    /// </summary>
+    [Parameter]
+    public bool CollapsedChildNav { get; set; } = false;
```

Comment:
Please rename to `CollapsedChildNavigation` to better convey meaning. With ...Nav people might think it is another element instead of a 'switch'